modules/virt-creating-configmap.adoc

```diff
@@ -66,7 +66,7 @@ data:
     "<rhv-operating-system>": "<vm-template>" <2>
 EOF
 ----
-<1> Optional: You can change the value of the `namespace` parameter.
+<1> Optional: You can change the namespace.
```

Comment:
Name can be changed too

Reply:
I know they can change the name but I would prefer that they not do this.